rust2/core/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "adbc_core"
+description = "Public abstract API, driver manager and driver exporter"
+version = { workspace = true }
+edition = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+
+[dependencies]
+arrow = { workspace = true }
+libloading = "0.8"
```

Comment:
For the moment we need to keep it here because we `impl From<libloading::Error> for Error` in `error.rs`.

But this raises an interesting question: how should we structure the workspace?

Currently, we have:
- A `core` crate that contains traits/structs for the API, the driver manager and the driver exporter
- A crate for each driver in `drivers/*`

Should we split the driver manager and the driver exporter into different crates?

One possible solution:
- Move the driver manager to `drivers/` (it's a driver after all)
- Keep the driver exporter into the core crate (because it does not introduce any new dependency and it's tightly coupled to the abstract API)

Reply:
Your suggestion sounds good to me.

Reply:
After trying to move the driver manager to its own crate I think that we should reconsider this. In doing so I had to publicly expose a lot of item (mostly related to FFI) which are just implementation details...

Alternatively, we could gate the driver manager behind a feature flag. Maybe we could do the same for the driver exporter even though it's less relevant because it does not introduce any additional dependency.